pkgs/development/mobile/androidenv/emulate-app.nix

```diff
@@ -1,16 +1,26 @@
 { composeAndroidPackages, stdenv, lib, runtimeShell }:
-{ name, app ? null
+{ name
+, app ? null
 , platformVersion ? "33"
 , abiVersion ? "armeabi-v7a"
 , systemImageType ? "default"
-, enableGPU ? false
-, extraAVDFiles ? []
+, enableGPU ? false # Enable GPU acceleration. It's deprecated, instead use `configOptions` below.
+, configOptions ? (
```

Comment:
would it make sense to turn this into an attribute set? Then it would compose better in cases where multiple overrides are stacked.

Reply:
@pbsds sorry for late reply! that actually makes sense, but it would be a little bit more complicated than what we currently have. In current code we just append the config option to the end of the config.ini file, acting like a `key = value` is just a row, but with attribute set we are demanding more information, which will lead to parsing config.ini file, then modify it and serialize it back. I'll try to test that out to see how bad it could get. thanks for the idea.

Reply:
```
nix-repl> lib.toINI {} { section.foo = "bar"; }  
"[section]\nfoo=bar\n"
```

https://noogle.dev/?selected=%22lib.generators.toINI%22&term=%22toini%22

Reply:
finally found some time to make it work. Let me know what do you think [367b6ae](https://github.com/NixOS/nixpkgs/pull/265504/commits/367b6ae0208ebd899b6b98d8801c2a32b1e81596)

Reply:
Aside from the typo in the git commit message it looks good to me :+1:

Reply:
cool. rebased and squashed.